daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/OrderingSpec.scala

```diff
@@ -326,6 +329,27 @@ class OrderingSpec
     Table("any", anys: _*),
   )
 
+  private val randomComparableValues: TableFor2[String, Gen[SValue]] = {
+    import com.daml.lf.value.TypedValueGenerators.{ValueAddend => VA}
+    implicit val ordNo
+      : Order[Nothing] = Order order [Nothing]((_: Any, _: Any) => sys.error("impossible"))
+    def r(name: String, va: VA)(sv: va.Inj[Nothing] => SValue) =
+      (name, va.injarb[Nothing].arbitrary map sv)
+    Table(
+      ("comparable value subset", "generator"),
+      Seq(
+        r("Int64", VA.int64)(SInt64),
+        r("Text", VA.text)(SText),
+        r("Int64 Option List", VA.list(VA.optional(VA.int64))) { loi =>
+          SList(loi.map(oi => SOptional(oi map SInt64)).to[FrontStack])
+        },
```

Comment:
`inj` gives us transaction library `Value`s for free, but not `SValue`s. We could go through the conversion instead, or define an associated `injSvalue` paired with each `ValueAddend` constructor we care to use. Of course for a small number of cases it's sufficient and safe to hand-write as done above.